qcodes/dataset/sqlite_base.py

```diff
@@ -1332,7 +1334,16 @@ def get_parameter_data(conn: ConnectionPlus,
         param_names = [param.name for param in paramspecs]
         types = [param.type for param in paramspecs]
 
```

Comment:
it's not related to this PR, but [the behavior from this line](https://github.com/QCoDeS/Qcodes/pull/1512/files#diff-c7bb6e3db6832c32d8ae8596d8991d78R1328) does not seem to be documented in the docstring.

This is the missing part that i've found in the DataSet method's docstring:
```
If no paramerers are supplied the values will
be returned for all parameters that are not them self dependencies.
```